### PR TITLE
Added activity logs to current editable models in publisher

### PR DESCRIPTION
**Pull request by jesilz**

### CODE REVIEW

File: app/controllers/activity_logs_controller.rb [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+  def new_activity_log_from_params
+    if params[:activity_log]
+      @activity_log = ActivityLog.new(activity_log_params)
+    else
+      @activity_log = ActivityLog.new
+    end
+  end
+
+  def activity_log_params
+    params.require(:activity_log).permit(:reviewer_ids, :action_performed, :message, :snapshot, :child)
```
Comment (rift137) [inline app/controllers/activity_logs_controller.rb]: These should be moved into the policy

File: app/views/important_dates/_menu.html.slim [hunk 2]
```diff
@@ -4,4 +4,5 @@
       = @important_date.title
     = menu_block(class: 'list-group-item') do |menu|
       = menu.add_item('Overview', default: true) { edit_important_date_path(@important_date, page: 'profile') }
+      = menu.add_item('History') { edit_important_date_path(@important_date, page: 'activity_logs') } if policy(@important_date).can_manage?(:activity_logs)
```
Comment (rift137) [inline app/views/important_dates/_menu.html.slim]: I think if you were to pass @importaint_date through as obj or something like that you would be able to combine all of the activity log partials into one shared file.

Comment (jesilz) [inline app/views/important_dates/_menu.html.slim]: agreed.. I just didn't know how to dynamically generate dynamic edit_*_path methods.

Comment (rift137) [inline app/views/important_dates/_menu.html.slim]: you should be able to use something like send("edit_#{obj}_path")

**Comment by rift137** [comment]
:+1:

**Pull request merged**